File: backend/src/main/java/com/gestion/blanchiment/dto/Userdto.java
package com.gestion.blanchiment.dto;




public class Userdto {

    	private String email;
	private String firstName;
	private String lastName;
	private Long phoneNumber;
	private String username;
	private Long authority;
	private boolean isenabled;
	private String password;
	
	public String getEmail() {
	    return email;
	}
	public void setEmail(String email) {
	    this.email = email;
	}
	public String getFirstName() {
	    return firstName;
	}
	public void setFirstName(String firstName) {
	    this.firstName = firstName;
	}
	public String getLastName() {
	    return lastName;
	}
	public void setLastName(String lastName) {
	    this.lastName = lastName;
	}
	
	public String getUsername() {
	    return username;
	}
	public void setUsername(String username) {
	    this.username = username;
	}
	public Long getPhoneNumber() {
	    return phoneNumber;
	}
	public void setPhoneNumber(Long phoneNumber) {
	    this.phoneNumber = phoneNumber;
	}
	public Long getAuthority() {
	    return authority;
	}
	public void setAuthority(Long authority) {
	    this.authority = authority;
	}

	
	public Userdto(String email, String firstName, String lastName, 
		Long phoneNumber, String username, Long authority) {
	    super();
	    this.email = email;
	    this.firstName = firstName;
	    this.lastName = lastName;
	    this.phoneNumber = phoneNumber;
	    this.username = username;
	    this.authority = authority;
	}
	public boolean isIsenabled() {
		return isenabled;
	}
	public void setIsenabled(boolean isenabled) {
		this.isenabled = isenabled;
	}
	public String getPassword() {
		return password;
	}
	public void setPassword(String password) {
		this.password = password;
	}
	
	
	
}

File: frontend/src/app/Model/userdto.ts
export class UserDto{

    private email :string;
     private firstName :string;
     private lastName :string;
     private phoneNumber:number;
     private authority:number;
     private isenabled:boolean;
  private username :string;
  private password :string;
  
    public constructor(email:string,firstName:string,lastName:string,phoneNumber:number,authority:number,isenabled:boolean,username :string,passowrd :string){
        this.email=email
        this.firstName=firstName
        this.lastName=lastName
        this.phoneNumber=phoneNumber
        this.authority=authority
        this.isenabled=isenabled
        this.password=passowrd
        this.username=username
      
    }
  }